<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(189, 189, 189);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="EncryptionBtn">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>87</y>
      <width>164</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string> 加密    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecryptionBtn">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>135</y>
      <width>164</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string> 解密    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="FinishedBtn">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>185</y>
      <width>164</width>
      <height>47</height>
     </rect>
    </property>
    <property name="text">
     <string> 文件管理</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchEdit">
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>95</y>
      <width>159</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>872</x>
      <y>87</y>
      <width>6</width>
      <height>576</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>87</y>
      <width>6</width>
      <height>576</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>125</y>
      <width>697</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="RightWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>873</x>
      <y>125</y>
      <width>200</width>
      <height>538</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="TopWidget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>1062</width>
      <height>74</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 240, 246);</string>
    </property>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>23</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Microsoft JhengHei&quot;;</string>
     </property>
     <property name="text">
      <string>昵称获取失败</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>824</x>
       <y>23</y>
       <width>80</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="UserPhonelabe">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>获取失败</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>987</x>
       <y>32</y>
       <width>12</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>1039</x>
       <y>32</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="UserIDlabel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名：空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_max">
     <property name="geometry">
      <rect>
       <x>1013</x>
       <y>33</y>
       <width>13</width>
       <height>9</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>976</x>
       <y>28</y>
       <width>6</width>
       <height>17</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LeftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>235</y>
      <width>165</width>
      <height>102</height>
     </rect>
    </property>
    <widget class="QPushButton" name="FinEnpBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>164</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>     已加密文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FinDepBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>164</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>     已解密文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="MidStaWidget">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>126</y>
      <width>698</width>
      <height>497</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QStackedWidget" name="BtnStaWidget">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>85</y>
      <width>698</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="OpenFileBtn">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="accessibleDescription">
       <string>批量删除</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>16</y>
        <width>53</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>6</y>
        <width>111</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>二维码</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>246</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>6</y>
        <width>111</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>16</y>
        <width>53</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_groupshare">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>127</x>
        <y>6</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>6</y>
        <width>111</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>16</y>
        <width>53</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>85</y>
      <width>1062</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>1037</x>
      <y>95</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>620</y>
      <width>895</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>131</y>
      <width>164</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>181</y>
      <width>164</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>231</y>
      <width>164</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>281</y>
      <width>164</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>332</y>
      <width>164</width>
      <height>6</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1084</width>
      <height>674</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/mainwindow/pictures/mainwindow_border.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>EncryptionBtn</zorder>
   <zorder>DecryptionBtn</zorder>
   <zorder>FinishedBtn</zorder>
   <zorder>SearchEdit</zorder>
   <zorder>RightWidget</zorder>
   <zorder>TopWidget</zorder>
   <zorder>LeftWidget</zorder>
   <zorder>MidStaWidget</zorder>
   <zorder>BtnStaWidget</zorder>
   <zorder>line_2</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>line</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_9</zorder>
   <zorder>line_10</zorder>
   <zorder>line_11</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
